components/UserTestimonies/ViewTestimony.tsx

```diff
@@ -113,6 +115,42 @@ const Author = styled<{ testimony: Testimony }>(({ testimony, ...props }) => {
   }
 `
 
+// https://stackoverflow.com/a/42234988/4531028
+function useOutsideClick(ref: RefObject<HTMLDivElement>, action: () => void) {
+  function handleClickOutside(event: MouseEvent) {
+    // TODO: type safety?
+    if (ref.current && !ref.current.contains(event.target as Node)) {
+      action()
+    }
+  }
+  document.addEventListener("mousedown", handleClickOutside)
+  return () => {
+    document.removeEventListener("mousedown", handleClickOutside)
+  }
+}
+
+const MoreButton = ({ children }: { children: React.ReactChild }) => {
```

Comment:
Hi @tommagnusson looks great! Below is our code of conduct - and I think your categories aptly capture the main points.

COMMUNITY GUIDELINES:
Stay on topic, use common courtesy, and be respectful of others.
You may not misrepresent your identity.
You may not post testimonies that are abusive, inflammatory, or otherwise inappropriate. 
You may not post spam or unnecessary repeated testimonies. 
You may not threaten or promote violence against an individual or a group of people.
You may not threaten or promote terrorism or violent extremism.
You may not engage in the targeted harassment of someone.
You may not promote violence against, threaten, or harass other people on the basis of race, ethnicity, national origin, sexual orientation, gender, gender identity, religious affiliation, age, or disability.
You may not use our service for any unlawful purpose or in furtherance of illegal activities. 
You may not publish or post other people's private information, including names, physical addresses, email addresses, and phone numbers.